Package.swift

```diff
@@ -1,50 +1,42 @@
 // swift-tools-version:5.4
+
 import PackageDescription
-let excludes = ["README.md"]
 
 let package = Package(
-    name: "ClientRuntime",
```

Comment:
We must not rename the packages yet. This is something we have to confirm with Smithy team.

Historically we have chosen `ClientRuntime` suffix which I don't necessarily agree with comparing with other SDKs. let's keep them as they are as of now so we only break customers once.

Reply:
@mtdowling is the person to go and confirm the names here and shorter is better for less symbol size.

Few suggestions
Runtime suffix
- SmithyRuntime
- AWSRuntime

Core suffix
- SmithyCore
- AWSCore

Reply:
I am rolling this name change back in this PR, but I'm also filing an issue on it because I think this should be explored prior to the 1.0.

Reply:
https://github.com/awslabs/smithy-swift/issues/438